en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md

```diff
@@ -16,40 +16,68 @@ If you do not want to configure this yourself, you can simply [get the project](
 
 ## Set up Kafka
 
-Before you begin, set up Kafka by following the instructions in [Setting up Kafka]({{base_path}}/reference/connectors/ksfka-connector/setting-up-kafka/).
+Before you begin, set up Kafka by following the instructions in [Setting up Kafka]({{base_path}}/reference/connectors/kafka-connector/setting-up-kafka/).
 
-## Configure the connector in WSO2 Integration Studio
+## Setting up the Integration Project
 
-Follow these steps to set up the Integration Project and the Connector Exporter Project. 
+1. Follow the steps in [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project.
 
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!}
+2. Create a new Kafka connection. 
+    1. Goto `Local Entries` -> `Connections` and click on the `+` sign. 
+    2. Select  `KafkaTransport` connector.
+        <img src="{{base_path}}/assets/img/integrate/connectors/kafka-conn-add-new-connection.png" title="Add new kafka connection" width="800" alt="Add new kafka connection"/>
 
-5. Create a new Kafka connection by selecting a particular operation.
+    3. Use following values can to create the connection. 
+        - Connection Name - KafkaConnection
+        - Connection Type - kafka
+        - Bootstrap Servers - localhost:9092 
+        - Key Serializer Class - org.apache.kafka.common.serialization.StringSerializer
+        - Value Serializer Class - org.apache.kafka.common.serialization.StringSerializer
+        - Pooling Enabled - false
 
-    <a href="{{base_path}}/assets/img/integrate/connectors/filecon10.png"><img src="{{base_path}}/assets/img/integrate/connectors/filecon10.png" title="working directory" width="800" alt="working directory"/></a>
+## Creating the Integration Logic
```

Comment:
Use sentence case and avoid `ing` in verbs for titles. https://learn.microsoft.com/en-us/style-guide/text-formatting/formatting-titles

Check and fix all places.